packages/flutter_tools/test/commands/test_test.dart

```diff
@@ -23,31 +23,37 @@ void main() {
     final String automatedTestsDirectory = fs.path.join('..', '..', 'dev', 'automated_tests');
     final String flutterTestDirectory = fs.path.join(automatedTestsDirectory, 'flutter_test');
 
-    testUsingContext('report nice errors for exceptions thrown within testWidgets()', () async {
-      Cache.flutterRoot = '../..';
-      return _testFile('exception_handling', automatedTestsDirectory, flutterTestDirectory);
-    });
-
-    testUsingContext('report a nice error when a guarded function was called without await', () async {
-      Cache.flutterRoot = '../..';
-      return _testFile('test_async_utils_guarded', automatedTestsDirectory, flutterTestDirectory);
-    });
-
-    testUsingContext('report a nice error when an async function was called without await', () async {
-      Cache.flutterRoot = '../..';
-      return _testFile('test_async_utils_unguarded', automatedTestsDirectory, flutterTestDirectory);
-    });
-
-    testUsingContext('report a nice error when a Ticker is left running', () async {
-      Cache.flutterRoot = '../..';
-      return _testFile('ticker', automatedTestsDirectory, flutterTestDirectory);
-    });
-
-    testUsingContext('report a nice error when a pubspec.yaml is missing a flutter_test dependency', () async {
-      final String missingDependencyTests = fs.path.join('..', '..', 'dev', 'missing_dependency_tests');
-      Cache.flutterRoot = '../..';
-      return _testFile('trivial', missingDependencyTests, missingDependencyTests);
-    });
+    // TODO(bkonyi): These tests all throw exceptions with unicode characters used for formatting,
```

Comment:
Can you mark them individually as skipped via [1]? That way, they'll get reported as skipped by the test runner. Also might be good to file an issue about this and attaching it to the TODO.

[1] https://github.com/flutter/flutter/blob/0350c9ecff293f1ca595577926a30b9758c239df/packages/flutter_tools/test/src/context.dart#L72

Reply:
to add to @goderbauer's comments: in general, please prefer using `skip: ...` rather than an if statement, because skipped tests count towards our technical debt, whereas if statements do not.

Reply:
Ah, didn't realize that was a parameter. Will do.